app/build.gradle

```diff
@@ -1,5 +1,12 @@
 apply plugin: 'com.android.application'
 
+allprojects {
+    repositories {
+        jcenter()
+        maven { url 'https://maven.google.com' }
```

Comment:
Does maven.google.com host OpenSource software? We need to enjure this is supported by f-droid.